### DISCUSSION TITLE
11/4 19:00~ 全体ミーティング　#1

### DISCUSSION THREAD

**Discussion by ayase-mstk** [42-pong/42-pong#1] [Meetings]
## 議題（アジェンダ）
- 前回話したチーム開発ルール（ひな型）の共有
- 課題読み合わせ
- 全体目標の決定（出来たら）
- 次回のタスク作成

**Comment by ayase-mstk** [comment]
何段階かのフェーズに分けて開発進める
（仮）
1. 認証などの機能のみある状態（ゲームのない状態）
2. ゲームができる機能（ソロゲーム）
3. トーナメント機能、マッチング機能
4. 追加モジュール

**Comment by ayase-mstk** [comment]
次回タスク

- 土台作り
  - branch命名規則, wiki移植
  - discord通知workflow作成
    - PRレビュワー割り当て通知
    - Discussion起票通知
  - 学習のためにコミットルールのひな型作成
    - https://tam-tam.co.jp/tipsnote/program/post16686.html
  
  - フォーマッターとリンターを先に決めとく

**Reply by habvi** [reply]
task link
- #4 
- #2 
- #3 
- #6
